test/unit/output/ferc1_test.py

```diff
@@ -185,6 +189,7 @@ def setUp(self):
         )
 
     def test_leafward_prop_undecided_children(self):
+        """Test leadward propagation with undecided children."""
```

Comment:
Were pre-commit checks yelling at you for not adding docstrings to these test methods? If so we should tell the check to ignore test docstrings.